server/controllers/user.js

```diff
@@ -1,5 +1,6 @@
 const { User } = require("../models/User");
 const Board = require("../models/Board");
```

Comment:
Use `const { Board }` here instead

Reply:
Future PR has dealt with this already 👍🏻